### DISCUSSION TITLE
Feature Request - Wyze Bulb Sleep Routine Disable/Enable

### DISCUSSION THREAD

**Discussion by rickscherer** [SecKatie/ha-wyzeapi#256] [Ideas]
It would be great if we could disable/enable the Sleep Routine capability of the Wyze Bulbs.

Unfortunately there is no way to use a Rule within the Wyze App, so I often forget to manually disable when leaving the house for an extended period -- and when we get home, our bedroom lights are on :)

**Comment by JoeSchubert** [comment]
I don't own any bulbs... What exactly is "the sleep capability"? Is it something that you toggle in the app?

**Comment by rickscherer** [comment]
Yep, simple On/Off toggle in the app. Once On there are settings to control
brightness/duration. Sleep Routines are per bulb or group.

**Comment by reidprichard** [comment]
I second and can help test. I have a sleep routine - specifically, a wakeup routine - that ramps my bedroom lamp on for the half hour leading up to my alarm. (Makes waking up much nicer - would recommend.) Anyway, sometimes I'll change my alarm, and if I forget to go into the Wyze app to change the routine as well I'll either A) wake up in darkness or B) wake up before my alarm because my lamp is on. It would be super nice to be able to make an automation that changes the wakeup routine start time based on my phone's next alarm time.

**Reply by rickscherer** [reply]
I've been using Node-RED in the meantime, I'm not using the native Wyze functionality but I've mimicked it to perform the wake up sequence.  You may be able to have your phone alarm trigger it.

This is very rudimentary... @ time turn on light, then every minute increase brightness by a count of 3. It checks each time to see if the light is on, if it's off then it stops (when I'm awake I manually turn off the light).
